### ISSUE TITLE
version for scala 2.12 ?

### ISSUE THREAD

**Issue by visox** [angelcervera/osm4scala#31]
Hi,

could you release some version for 2.12 ?
i dont see any here
https://mvnrepository.com/artifact/com.acervera.osm4scala/osm4scala-core

thx

**Comment by angelcervera** [comment]
No idea why only the 2.11 version is published in the Sonatype repo.
I will check it and I will publish the 2.13 version as well.

In the meantime, you can use the Bintray repo, as the README explains:

```
resolvers += "osm4scala repo" at "http://dl.bintray.com/angelcervera/maven"
libraryDependencies += "com.acervera.osm4scala" %% "osm4scala-core" % "1.0.1"
```

**Comment by visox** [comment]
yes i did that, thx

**Comment by visox** [comment]
also add 2.13
http://dl.bintray.com/angelcervera/maven/com/acervera/osm4scala/

**Comment by angelcervera** [comment]
@visox https://bintray.com/angelcervera/maven/osm4scala/1.0.3
Version 1.0.3 released for Scala 2.11, 2.12 and 2.13

Waiting for response from Sonatype to be able to sync all versions into the central repository.

**Comment by angelcervera** [comment]
https://search.maven.org/search?q=osm4scala
